src/frontends/tensorflow/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenVINO TensorFlow Frontend
+
+The TensorFlow Frontend (TF FE) is a C++ based OpenVINO Frontend component that is responsible for reading and converting a TensorFlow model to an `ov::Model` object
+that further can be serialized into the Intermediate Representation (IR) format.
+
+```mermaid
+flowchart BT
+    model[(model1.pb)]
+    style model fill:#427cb0
+    model2[(model2.pb)]
+    style model2 fill:#427cb0
+    ovtf(OpenVINO Integration with TensorFlow)
+    style ovtf fill:#ffffc2
+    tf_fe(TensorFlow Frontend)
+    style tf_fe fill:#ee9a4d
+    fem(Frontend Manager)
+    mo(Model Optimizer)
+    model --> ovtf --> tf_fe
+    model2 --> mo --> fem --> tf_fe
+    click ovtf "https://github.com/openvinotoolkit/openvino_tensorflow"
+```
+
+Currently, it is only used by [OpenVINO Integration with TensorFlow Frontend](https://github.com/openvinotoolkit/openvino_tensorflow).
+Model Optimizer for now relies on the legacy TensorFlow Frontend developed in Python.
+
+TensorFlow Frontend uses [the common coding style rules](../../../docs/dev/coding_style.md).
+
+## Key contacts
+
+People from the [openvino-tf-frontend-maintainers](https://github.com/orgs/openvinotoolkit/teams/openvino-tf-frontend-maintainers) have the rights to approve and merge PRs to the TensorFlow Frontend component.
+They can assist with any questions about the component.
+
+## Components
+
+The structure of OpenVINO TensorFlow Frontend sources includes the following directories:
+
+* [include](./include) is a public frontend API.
+* [src](./src/) folder contains the sources of the component.
+* [tests](./tests) cover the internal transformations.
+
+## Architecture
+
+OpenVINO TensorFlow Frontend uses [TensorFlow Protobuf files](./src/proto) for reading and parsing TensorFlow model formats.
+
+The conversion of the TensorFlow operation requires either one pass using [Loaders](./src/op) or two transformation passes
+using [Loaders](./src/op) and [Internal Transformation](./src/helper_transforms).
```

Comment:
It is not clear what does it mean without reading that part about architecture. It is better to move the complete architecture article here. It is not too big to be separated to a different page.

Reply:
Agree, let us do it.

Reply:
Done